src/Components/Button.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import styled from "styled-components";
+
+const Button = ({
+  onClick,
+  children,
+  subText,
+  bgcolor,
+  color,
+  width,
+  weight,
+}) => {
+  return (
+    <ButtonStyle
+      onClick={onClick}
+      bgcolor={bgcolor}
+      color={color}
+      width={width}
+      weight={weight}
+    >
+      <span className="text">{children}</span>
+      <p clasName="subtext">{subText}</p>
+    </ButtonStyle>
+  );
+};
+
+const ButtonStyle = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: ${(props) => props.bgcolor || "black"};
+  color: ${(props) => props.color || "white"};
+  width: ${(props) => props.width || "100%"};
```

Comment:
```suggestion
  background: ${({ bgcolor }) => bgcolor || "black"};
  color: ${({ color }) => color || "white"};
  width: ${({ width }) => width || "100%"};
```
함수 파라미터 자리에서 구조분해 할당 가능합니다